ycmd/completers/cpp/clang_completer.py

```diff
@@ -185,11 +231,35 @@ def _GoToImprecise( self, request_data ):
       raise RuntimeError( 'Can\'t jump to definition or declaration.' )
     return _ResponseForLocation( location )
 
+  def _GetSemanticInfo( self, request_data, func, reparse = True ):
+    filename = request_data[ 'filepath' ]
+    if not filename:
+      raise ValueError( INVALID_FILE_MESSAGE )
+
+    flags = self._FlagsForRequest( request_data )
+    if not flags:
+      raise ValueError( NO_COMPILE_FLAGS_MESSAGE )
+
+    files = self.GetUnsavedFilesVector( request_data )
+    line = request_data[ 'line_num' ]
+    column = request_data[ 'column_num' ]
+
+    type = getattr( self._completer, func )(
+        ToUtf8IfNeeded( filename ),
+        line,
+        column,
+        files,
+        flags,
+        reparse)
+
+    if type == "":
```

Comment:
Just `if type`. But don't use the name `type`, it's the name of a Python built-in function and you're overwriting it. You can see the blue color in the syntax-highlighting here..